src/main/java/eu/bbmri/eric/csit/service/negotiator/database/model/Negotiation.java

```diff
@@ -76,6 +76,8 @@ public class Negotiation extends AuditEntity {
   @Column(columnDefinition = "jsonb")
   private String payload;
 
+  private String status;
```

Comment:
Why have you added this field?

Reply:
I' ve never added it, it's strange that it is marked as mine in the review :-)

Reply:
oh okay

Reply:
Fixed in 9e79a341c22c9d82b6f49666b7b974893587a33c